packages/odyssey-react-mui/src/Field.tsx

```diff
@@ -118,7 +129,7 @@ const Field = ({
     <MuiFormControl
       component={fieldType === "group" ? "fieldset" : "div"}
       disabled={isDisabled}
-      error={Boolean(errorMessage)}
+      error={Boolean(errorMessage) || Boolean(errorMessagesList)}
```

Comment:
This should probably check for the existence of 1+ items rather than the array itself.